### DISCUSSION TITLE
Dual range slider

### DISCUSSION THREAD

**Discussion by hassanbhattignomen** [formkit/formkit#536] [Ideas]
### Describe the feature

dual range slider use for some specific search like pricing section user want to explore products between 50 and 100 or 70 and 150 so when dual range slider min and max values settled search results appear. and its best for any industry with search. 
<img width="703" alt="Screenshot 2022-11-22 at 2 01 51 PM" src="https://user-images.githubusercontent.com/98885223/203271271-580e251e-4646-4826-8d09-0df5d6686112.png">
as an example see attached image. 
I think it is useful search filter so please consider it.

**Comment by justin-schroeder** [comment]
Thanks.

**Comment by justin-schroeder** [comment]
If we do eventually implement this — we think it should include options for showing more data along with the slider. A slider with just min/max seems almost like a UX regression over just having a min/max value unless we actually provide some kind of additional information that is spatially represented:

![image](https://user-images.githubusercontent.com/1950463/212356379-afb3ebd8-5dcd-4f6e-af5a-d895caa42bdb.png)

**Comment by andrew-boyd** [comment]
Hey-o! This is almost ready to go. Buttoning up Tailwind styling on it and would love to get some eyes on the draft docs page if anyone's following this discussion and wants to take a look. https://formkit-com-git-feature-slider-formkit.vercel.app/inputs/slider

**Comment by andrew-boyd** [comment]
This is live now: https://formkit.com/inputs/slider
